src/main/ui/package.json

```diff
@@ -23,7 +23,8 @@
     "react": "^15.4.2",
     "react-bootstrap": "^0.30.7",
     "react-dom": "^15.4.2",
-    "react-router": "^3.0.2"
+    "react-router": "^3.0.2",
+    "react-select" : "*"
```

Comment:
Did you do a regular `npm install --save-dev`` for this? I wonder why it's showing * instead of a minimum version number.